docs/usage/getting-started/use-cases.md

```diff
@@ -52,12 +52,12 @@ A simplified example:
 ### Custom dependency extraction
 
 Renovate supports 60+ types of package files.
-Not all dependencies are detected by default, this can be because:
+Only some dependencies are found by default, this can be because:
```

Comment:
```suggestion
By default, Renovate finds most dependencies, but there are exceptions.
This can be because:
```

I'm not sure what wording is best here:

- The current wording
- The wording in the suggestion

I like that the suggestion shows that often it "just works". Let me know what you like best. 😉